store/demo/dev/10111__task_dag.sql

```diff
@@ -0,0 +1,29 @@
+-- task_dag for task dependency
+
+INSERT INTO
+    task_dag (
+        id,
+        from_task_id,
+        to_task_id
+    )
+VALUES
+    (
+        11101,
+        11018,
```

Comment:
does this reflect what we discussed from/to yesterday?

Reply:
No. I'll switch from&to in another PR.